silk/node/fifteen_four_dev_board.py

```diff
@@ -137,10 +142,10 @@ def __init__(self, wpantund_verbose_debug=False, sw_version=None):
 
         # Check what platform Silk is running on.  If it is an ARM device,
         # allow up to 2 minutes for wpantund to start.
-        if "armv7l" in subprocess.check_output("uname -m", shell=True):
-            self.wpantund_start_time = 120  # seconds
-        else:
-            self.wpantund_start_time = 30
+        #if "armv7l" in subprocess.check_output("uname -m", shell=True):
+        #    self.wpantund_start_time = 120  # seconds
+        #else:
```

Comment:
Please add a TODO here. We will fix this later.

Reply:
Please remove commented code and comments and add TODO with change/issue ID link created in github. we should keep detail information in the issue link